libraries/chain/db_block.cpp

```diff
@@ -632,14 +665,19 @@ processed_transaction database::_apply_transaction(const signed_transaction& trx
    transaction_evaluation_state eval_state(this);
    const chain_parameters& chain_parameters = get_global_properties().parameters;
    eval_state._trx = &trx;
-
+   
+   if( head_block_time() > HARDFORK_CORE_1285_TIME )
+   {
+      verify_custom_authorities(trx);
```

Comment:
So, you here test if a `trx` may be signed by a custom id. If not it fails, but if it is signed by a custom authority, how will the `trx` pass the next test in line #678 with `trx.verify_authority`?

Reply:
Also, shouldn't this still allow verification by *regular* signature by active key?

Reply:
>  how will the trx pass the next test in line 678 with trx.verify_authority?
I will pass the trx.verify_authority as it did before custom authority implementation. 
Custom authority is just additional check it doesn't affect authority validations.

Reply:
Done. Please check transaction.cpp file changes.